src/neuroglancer/datasource/brainmaps/backend.ts

```diff
@@ -207,6 +211,100 @@ export class MeshSource extends ParameterizedMeshSource<MeshSourceParameters> {
     return makeRequest(parameters['instance'], httpCall, cancellationToken)
         .then(response => decodeFragmentChunk(chunk, response));
   }
+
+  filterFragments(fragmentIds: string[]|null, clipBounds: Bounds) {
```

Comment:
Instead of needing a separate filterFragments method, the fragments could just be filtered when the manifest is downloaded.